dlc_developer_config.toml

```diff
@@ -27,6 +27,12 @@ trcomp_mode = false
 # Please only set it to True if you are preparing a Benchmark related PR
 # Do remember to revert it back to False before merging any PR (including Benchmark dedicated PR)
 benchmark_mode = false
+# Set deep_canary_mode to true to simulate Deep Canary Test conditions on PR for all frameworks in the
+# build_frameworks list below. This will cause all image builds and non-deep-canary tests on the PR to be skipped,
+# regardless of whether they are enabled or disabled below.
+# Set graviton_mode to true to run Deep Canaries on Graviton images.
+# Do remember to revert it back to false before merging any PR.
+deep_canary_mode = true
```

Comment:
is this going to be the default? I think the default is meant to be false. If so, can we update this to false? Understand the PR quick check will fail with the new additions, with the understand that this will become the new template.

Reply:
Oops, yes, this was a miss. It should be `false` by default so that CI tests work as usual on non-canary PRs.